Source/WebKit/NetworkProcess/NetworkLoadChecker.cpp

```diff
@@ -213,7 +213,7 @@ ResourceError NetworkLoadChecker::validateResponse(const ResourceRequest& reques
 
     if (m_options.mode == FetchOptions::Mode::Navigate || m_isSameOriginRequest) {
         if (m_options.mode == FetchOptions::Mode::Navigate && m_parentOrigin) {
-            if (auto error = performCORPCheck(m_parentCrossOriginEmbedderPolicy, *m_parentOrigin, m_url, response, ForNavigation::Yes, m_networkResourceLoader.get(), originAccessPatterns()))
+            if (auto error = performCORPCheck(m_parentCrossOriginEmbedderPolicy, *m_parentOrigin, m_url, response, ForNavigation::Yes, RefPtr { m_networkResourceLoader.get() }.get(), originAccessPatterns()))
```

Comment:
Should `performCORPCheck(...)` be changed to accept a `RefPtr&`?

Reply:
I don't think we need to do that. The current rule is that caller of a function needs to keep all of its arguments alive. We don't necessarily need to have each function take a RefPtr.